scripts/beaker/ladder-launch.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -ex
+
+NUM_NODES=$1
+shift
+
+if [[ $NUM_NODES -eq 1 ]]; then
+  MULTI_NODE_ARGS=""
+  COMMAND="scripts/beaker/ladder.sh localhost ${NUM_NODES} 0 $*"
+else
+  MULTI_NODE_ARGS="--replicas ${NUM_NODES} --leader-selection --host-networking --propagate-failure --propagate-preemption --synchronized-start-timeout 10m"
+  COMMAND="scripts/beaker/ladder.sh \$BEAKER_LEADER_REPLICA_HOSTNAME ${NUM_NODES} \$BEAKER_REPLICA_RANK $*"
+fi
+
+gantry run \
+  --workspace ai2/OLMo-training \
+  --task-name ladder \
+  --description "OLMo ladder with $*" \
+  --priority normal \
+  --preemptible \
+  --beaker-image shanea/olmo-torch2.2-gantry \
```

Comment:
Consider upgrading this to `petew/olmo-torch23-gantry` or `shanea/olmo-torch23-gantry` (untested but has `importlib_resources`)

Reply:
Done in c295ceae2a0777fab5e07476dbee1b71a6f2fa4f, though I have a vague memory that I tried this and there was a problem with it.